src/babylon.scene.ts

```diff
@@ -3236,6 +3236,24 @@ module BABYLON {
             return index;
         }
 
+        /**
+         * Removes the given reflection probe from this scene.
+         * @param toRemove The reflection probe to remove
+         * @returns The index of the removed reflection probe
+         */
+        public removeReflectionProbe(toRemove: ReflectionProbe): number {

```

Comment:
We need to remove any dependencies from the same so can you augment it like we did here:
https://github.com/BabylonJS/Babylon.js/blob/master/src/Physics/babylon.physicsEngineComponent.ts

This is for the upcoming ESNext feature

Reply:
If I understood well, I moved these functions like the reflection probes are scene components
Kept the "AbstractScene" augmentations as it was done as this before
Am I right?